### DISCUSSION TITLE
Add support to use with Remote Desktop Connection

### DISCUSSION THREAD

**Discussion by viethung9at1** [LizardByte/community#322] [Sunshine] [answered]
Hi Sunshine team,
Can you please add the ability to stream the screen from RDP (I prefer using with RDP because it has many supports to redirect the devices). I can do it with Parsec but with Sunshine + Moonlight, I can't

Thank a lot!

**Comment by ReenigneArcher** [comment, marked as answer]
Not possible with the Desktop Duplication API.
